src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/index.tsx

```diff
@@ -0,0 +1,175 @@
+import { Box, Text, Tip } from 'grommet';
+import React, { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { InputBar } from 'components/Chat/SharedChatComponents/InputBar';
+import { ExitCrossSVG } from 'components/SVGIcons/ExitCrossSVG';
+import { QuestionMarkSVG } from 'components/SVGIcons/QuestionMarkSVG';
+import { converse } from 'utils/window';
+import { InputDisplayNameIncorrectAlert } from './InputDisplayNameIncorrectAlert/index';
+
+const messages = defineMessages({
+  closeButtonTitle: {
+    defaultMessage: 'Click this button to close the overlay.',
+    description: 'A title describing the close button action',
+    id: 'components.InputDiplayNameOverlay.closeButtonTitle',
+  },
+  inputAnonymousKeywordForbiddenAlertMessage: {
+    defaultMessage: 'Keyword "anonymous" is not allowed.',
+    description:
+      'An alert message explaining why the entered display name is invalid.',
+    id: 'components.InputDisplayNameOverlay.inputAnonymousKeywordForbiddenAlertMessage',
+  },
+  inputDisplayNameInformative: {
+    defaultMessage:
+      "The display name is the pseudonym you will be authentificated with on this live. You won't be able to change during the live. Other participants will see you with this name. Instructor will however be able to see your geniune identity if you have previously identified yourself with the LTI.",
```

Comment:
```suggestion
      "The display name is the pseudonym you will be authenticated with on this live. You won't be able to change it during the live. Other participants will see you with this name. The instructor will however be able to see your genuine identity if you have previously identified yourself with the LTI.",
```

Reply:
done in commit [d3c2a0](https://github.com/openfun/marsha/pull/1276/commits/d3c2a037d1f2b70d6fd332d3c8951c3bea0df3a5)